packages/model-viewer/src/features/scene-graph/sampler.ts

```diff
@@ -202,7 +193,10 @@ export class Sampler extends ThreeDOMElement implements SamplerInterface {
     const sampler = this[$sourceSampler];
     if (sampler != null) {
       if (isValidSamplerValue(property, value)) {
-        sampler[property] = value;
+        // Rotation, repeat, and offset are not defined in GltfSampler.
+        if (['rotation', 'repeat', 'offset'].indexOf(property) == -1) {
```

Comment:
How about simplify this until we don't need the comment: `property !== 'rotation' && property !== 'repeat'...`